test/Sema/immutability.swift

```diff
@@ -61,6 +61,12 @@ class FooClass {
   mutating            // expected-error {{'mutating' isn't valid on methods in classes or class-bound protocols}} {{3-12=}}
   func baz() {}
 
+  nonmutating         // expected-error {{'nonmutating' isn't valid on methods in classes or class-bound protocols}} {{3-15=}}
```

Comment:
I guess this is technically a source compatibility break -- I'm fine with taking it, but if there's fallout we might have to make it conditional on `-swift-version 5` (which hasn't been plumbed through yet).